### PR TITLE
Implement Blank class to allow for optional Person fields

### PR DESCRIPTION
**Pull request by flyingsalsa**

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/AddCommandParser.java [hunk 1]
```diff
@@ -39,16 +40,32 @@ public AddCommand parse(String args) throws ParseException {
...
+        } else {
+            email = new Email(new Blank());
+        }
+        Address address;
+        if (arePrefixesPresent(argMultimap, PREFIX_ADDRESS)) {
+            address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
+        } else {
+            address = new Address(new Blank());
+        }
+
```
Comment (Kitty-001) [inline src/main/java/seedu/address/logic/parser/AddCommandParser.java:68]: I feel the modifications to ParserUtil is enough to handle these. Maybe can do something like:
```
Phone phone = ParserUtil.parsePhone(argumentMultimap.getValue(PREFIX_PHONE).orElse(""));
```

File: src/test/java/seedu/address/model/person/EmailTest.java [hunk 2]
```diff
@@ -10,7 +10,7 @@ public class EmailTest {
 
     @Test
     public void constructor_null_throwsNullPointerException() {
-        assertThrows(NullPointerException.class, () -> new Email(null));
+        assertThrows(NullPointerException.class, () -> new Email((String) null));
```
Comment (Kitty-001) [inline src/test/java/seedu/address/model/person/EmailTest.java:13]: What is this for?

**Review by Kitty-001 — COMMENTED** [review]
LGTM!

File: src/test/java/seedu/address/model/person/EmailTest.java (hunk 2 above)
Comment (flyingsalsa) [reply src/test/java/seedu/address/model/person/EmailTest.java:13]: to call the correct constructor.

File: src/main/java/seedu/address/logic/parser/AddCommandParser.java (hunk 1 above)
Comment (flyingsalsa) [reply src/main/java/seedu/address/logic/parser/AddCommandParser.java:68]: Hmm i didnt realise it was an optional

**Pull request merged**